styles/dialog.css

```diff
@@ -66,9 +66,6 @@ ai-dialog > ai-dialog-header > button {
   padding: 0;
   cursor: pointer;
 }
-ai-dialog > ai-dialog-header > button:focus {
```

Comment:
Do you want me to get rid of the whole https://github.com/0x210F/aurelia-dialog/commit/017b5bf4555ca07e0fca1f9186728a78d14e4f4d commit or only the changes that affect the .css files?